addons/crm_iap_lead_enrich/__init__.py

```diff
@@ -2,13 +2,3 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from . import models
-
-from odoo.api import Environment, SUPERUSER_ID
-
-
-def _synchronize_cron(cr, registry):
```

Comment:
Why is this removed exactly?

One could still set the parameter to "manual" before installing this module, correct?